test/main_test.js

```diff
@@ -1,158 +1,198 @@
+          done();
           break;
       }
-    }
-  })
-  it('Test join, joinOrCreateRoom and exit room', (done) => {
-    var player2Id
-    var ws2 = new WebSocket('ws://localhost:3000');
+    };
+  });
+  it("Test join, joinOrCreateRoom and exit room", (done) => {
+    var player2Id;
+    var ws2 = new WebSocket("ws://localhost:3000");
     ws2.onopen = function () {
-      console.log('client 2 is open')
-    }
+      console.log("client 2 is open");
+    };
     ws2.onmessage = function (e) {
       jsonObj = JSON.parse(e.data);
       switch (jsonObj.type) {
-        case 'register':
-          expect(jsonObj.playerId).to.be.string
-          var playerKey = server.edgeMultiplay.lobby.nameMap[jsonObj.playerId]
-          expect(playerKey).not.undefined
-          var playerConnection = server.edgeMultiplay.lobby.connectedClients[playerKey]
-          expect(playerConnection.address).equal(ws2.address)
-          expect(playerConnection.port).equal(ws2.port)
+        case "register":
+          expect(jsonObj.playerId).to.be.string;
+          var playerKey = edgeMultiplay.lobby.nameMap[jsonObj.playerId];
+          expect(playerKey).not.undefined;
+          var playerConnection =
+            edgeMultiplay.lobby.connectedClients[playerKey];
+          expect(playerConnection.address).equal(ws2.address);
+          expect(playerConnection.port).equal(ws2.port);
           // join room
-          player2Id = jsonObj.playerId
-          joinReq = client_util.joinRoomRequest(roomId, 'Player2', player2Id, 0);
+          player2Id = jsonObj.playerId;
+          joinReq = client_util.joinRoomRequest(
+            roomId,
+            "Player2",
+            player2Id,
+            0
+          );
           ws2.send(JSON.stringify(joinReq));
           break;
-        case 'roomJoin':
-          expect(server.edgeMultiplay.lobby.rooms.size).equal(1)
-          expect(server.edgeMultiplay.lobby.availableRooms.size).equal(0)
-          expect(server.edgeMultiplay.lobby.fullRooms.size).equal(1)
-          expect(jsonObj.room.roomMembers.length).equal(2)
-          if (jsonObj.room.roomMembers[1].playerName === 'Player2-repeat') {
-            ws2.close()
-            setTimeout(() => { done() }, 100);
+        case "roomJoin":
+          expect(edgeMultiplay.lobby.rooms.size).equal(1);
+          expect(edgeMultiplay.lobby.availableRooms.size).equal(0);
+          expect(edgeMultiplay.lobby.fullRooms.size).equal(1);
+          expect(jsonObj.room.roomMembers.length).equal(2);
+          if (jsonObj.room.roomMembers[1].playerName === "Player2-repeat") {
+            ws2.close();
+            setTimeout(() => {
+              done();
+            }, 100);
           }
           break;
-        case 'gameStart':
-          var exitRoomReq = {}
-          exitRoomReq.type = 'ExitRoom'
-          exitRoomReq.roomId = roomId
-          exitRoomReq.playerId = player2Id
+        case "gameStart":
+          var exitRoomReq = {};
```

Comment:
does javascript not require semicolons?

Reply:
So I am using a new plugin called prettier and it just updates the file according to ES6.
semi colons are considered to be clean, I guess.